internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -415,21 +415,22 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{RequeueAfter: shortWait}, nil
 	}
 
-	// We want to ensure we can render all of our composed resources before we
-	// apply any of them. We prefer to avoid creating or updating any composed
-	// resources if we know we won't be able to create or update all of them.
+	// We optimistically render all composed resources that we are able to with
+	// the expectation that any that we fail to render will subsequently have
+	// their error corrected by manual intervention or propagation of a required
+	// input.
 	refs := make([]corev1.ObjectReference, len(tas))
 	cds := make([]resource.Composed, len(tas))
 	for i, ta := range tas {
 		cd := composed.New(composed.FromReference(ta.Reference))
+		refs[i] = *meta.ReferenceTo(cd, cd.GetObjectKind().GroupVersionKind())
```

Comment:
This isn't going to work as we have to add the reference after render -- I am likely going to use a different var to track if there were resources we were unable to render. Update coming.